src/ProductView/TabBar/QueryInterface/QueryInterfaceAttributeList/QueryInterfaceAttribute/QueryInterfaceAttribute.scss

```diff
@@ -0,0 +1,4 @@
+input.form-control[readonly] {
```

Comment:
@friedow I disagree with you on this point. IMHO the contrast between the "undefined"-grey and the blue of the "normal" attribute is too small (see attached screenshot) and this could lead to usability issues. 
![image](https://cloud.githubusercontent.com/assets/9576848/24589743/f75091f0-17df-11e7-84f8-2bbc26a755f4.png)